packages/dialtone-vue2/components/link/link_constants.js

```diff
@@ -2,21 +2,36 @@ export const DANGER = 'danger';
 export const WARNING = 'warning';
 export const SUCCESS = 'success';
 export const MUTED = 'muted';
-export const INVERTED = 'inverted';
 export const MENTION = 'mention';
-export const LINK_VARIANTS = ['', DANGER, WARNING, SUCCESS, MUTED, INVERTED, MENTION];
+export const LINK_VARIANTS = ['', DANGER, WARNING, SUCCESS, MUTED, MENTION];
 
 export const LINK_KIND_MODIFIERS = {
   default: '',
   warning: 'd-link--warning',
   danger: 'd-link--danger',
   success: 'd-link--success',
   muted: 'd-link--muted',
-  inverted: 'd-link--inverted',
```

Comment:
Would moving it out of here into new inverted prop break current use of inverted?

Reply:
Yes.. but also I think it's pretty confusing to still have it in there. Might be worth doing the migration.

Reply:
Yes, it would be a breaking change. I didn't find any use in firespotter though.